src/helpers/GiveRedemptionBalanceHelper.ts

```diff
@@ -97,50 +114,60 @@ export const getMockRedemptionBalancesAsync = async ({ address, networkID, provi
   };
 };
 
-/*
-  With the old YieldDirector contract hooked to MockSohm this will no longer work
-  but it will work with the new YieldDirector version that indexes event topics
-*/
+/**
+ * Gets the number of active donations (roughly, determining active donations is tricky)
+ * to a specific recipient's address
+ * @param address Ethereum address of the target recipient
+ * @param networkID ID number of the network the user is currently connected to
+ * @param provider Ethereum network provider object
+ * @returns Number of active donations to the provided address
+ */
 export const getDonorNumbers = async ({ address, networkID, provider }: IBaseAddressAsyncThunk) => {
-  // Addresses in EVM events are zero padded out to 32 characters and are lower case
-  // This matches our inputs with the data we expect to receive from Ethereum
-  const zeroPadAddress = ethers.utils.hexZeroPad(address, 32);
-
-  const givingContract = new ethers.Contract(addresses[networkID].GIVING_ADDRESS as string, OlympusGiving, provider);
-
-  // creates a filter looking at all Deposited events on the YieldDirector contract
-  const filter = {
-    address: addresses[networkID].GIVING_ADDRESS,
-    fromBlock: 1,
-    toBlock: "latest",
-    topics: [ethers.utils.id("Deposited(address,address,uint256)"), null, zeroPadAddress], // hash identifying Deposited event
-  };
+  const donationsToAddress = [];
+
+  if (!(addresses[networkID] && addresses[networkID].GIVING_ADDRESS)) {
```

Comment:
Exit the function

Reply:
same question as a few comments ago, doesn't it already? After it console.logs it just jumps straight to the return